test/middlewares/find.jsx

```diff
@@ -0,0 +1,45 @@
+import Test from '../../src/legit-tests'
+import {Find} from '../../src/middleware'
+import { expect } from 'chai'
+
+import TestComponent from '../components/component'
+import TinyComponent from '../components/tiny-component'
+
+describe('Find middleware', () => {
+
+  it('should find div', () => {
+    Test(<TestComponent/>)
+    .find('div')
+    .test(function() {
+      expect(this.elements.div.props.children).to.be.equal(undefined)
```

Comment:
@zackify Is `test` method even necessary?? The code seems a bit convoluted. `element` should be prefered of writing tests. Choose the elements you want to test in the elements args and write tests on them.